overlord/snapstate/refresh.go

```diff
@@ -156,18 +156,22 @@ func (err *BusySnapError) PendingSnapRefreshInfo() *userclient.PendingSnapRefres
 
 // Error formats an error string describing what is running.
 func (err *BusySnapError) Error() string {
+	pids := make([]string, 0, len(err.pids))
+	for _, p := range err.pids {
+		pids = append(pids, fmt.Sprintf("%d", p))
+	}
```

Comment:
nitpick: you could use `strutil.IntsToCommaSeparated()`

Reply:
ah, great suggestion, I didn't know we have it, thanks!